EIPS/eip-4950.md

```diff
@@ -0,0 +1,86 @@
+---
+eip: 4950
+title: Entangled Tokens
+description: ERC-721 extension with two tokens minted that are tied together
+author: Victor Munoz (@victormunoz)
+discussions-to: https://ethereum-magicians.org/t/entangled-tokens/8702
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-03-28
+requires: 721
+---
+
+## Abstract
+This EIP standardizes an interface for creating entangled pairs of tokens. An entangled pair of tokens give his owners the right to operate with this common wallet.

```

Comment:
Sorry to push back on this again, but I still don't find this abstract very clear. Do you mean something like:

```suggestion
This EIP standardizes an interface for creating entangled pairs of tokens. An entangled pair of tokens are always owned by the same wallet, and can be operated on by the same owner.

```

Reply:
This correction is wrong.
Entangled tokens are minted by the same contract, but will later be owned by two different addresses.
These tokens allow its owners to transact with the budget of the wallet that minted them.
How can I write this in a simple sentence?

Reply:
> allow its owners to transact with the budget of the wallet that minted them

I just don't see how this is technically possible, unless budget and/or wallet mean something different than what I'm used to.

Reply:
> I just don't see how this is technically possible, unless budget and/or wallet mean something different than what I'm used to.

By budget I mean the ERC20 tokens that the contract owns. And wallet is the address of the contract.
Specifically, the contract (EIP-4950), when published (deployed), gets an address. So you can send ERC20 tokens to this address. And you can also make this address to be the owner of some external NFTs (ERC721 token). These tokens are the "budget" of the contract wallet. And the overall idea of this EIP is that this contract can also mint tokens (entangled) and the owners of this entangled tokens will be able to operate with the contract budget,

Reply:
Ah! I think I might understand now. Let me know if I have this correctly:

> This EIP defines an interface for delegating control of a smart contract wallet to pairs of users using entangled EIP-721 non-fungible tokens.

If that's correct, I think it'd make a much more understandable abstract.